packages/wallet-core/src/tests/direct-funder.test.ts

```diff
@@ -167,7 +171,7 @@ describe('cranking', () => {
         ]
       }
     });
-    const zeroOutcome = initialize(signStateHelper(zeroOutcomeState, 'A'), 1);
+    const zeroOutcome = {...initialize(signStateHelper(zeroOutcomeState, 'A'), 1), approved: true};
```

Comment:
💡 should `initialize` accept an `approved` flag?

Reply:
I think that's a good idea 427adccc0